src/lib/operators/export.cpp

```diff
@@ -22,6 +23,18 @@ const std::string& Export::name() const {
   return name;
 }
 
+std::string Export::description(DescriptionMode description_mode) const {
+  const auto separator = (description_mode == DescriptionMode::SingleLine ? ' ' : '\n');
+
+  auto stream = std::stringstream{};
```

Comment:
What was the reason for using a stringstream instead of string manipulation?

Reply:
No specific reason, basically just copied it from elsewhere 😅